### ISSUE TITLE
Installing kelvin as a service (the correct way)

### ISSUE THREAD

**Issue by EmilyBjoerk** [stefanwichmann/kelvin#46]
I went through and installed kelvin as a service the "correct" way: service specific user (that cannot login), located in the recommended location for unmanaged software not built by the admin and auto started on boot. I found the systemd file under /etc after I wrote this (ﾉ≧∇≦)ﾉ ﾐ ┸━┸.

Feel free to do as you please with. I'm posting it here as I already went through the motions.

```
# Fetch release
wget https://github.com/stefanwichmann/kelvin/releases/download/v1.1.9/kelvin-linux-arm-v1.1.9.tar.gz -O /tmp/kelvin-arm.tar.gz

# Create user to run as
sudo adduser --system --group --shell /bin/nologin --no-create-home --home /opt/kelvin kelvin

# Install
sudo mkdir -p /opt/kelvin
cd /opt/kelvin
sudo tar -xvzf /tmp/kelvin-arm.tar.gz
sudo mv kelvin-linux-arm*/* .
sudo rmdir kelvin-linux-arm*
sudo chown -R kelvin:kelvin /opt/kelvin

# Create service file for systemd 
sudo -e /etc/systemd/system/kelvin.service
```

/etc/systemd/system/kelvin.service:
```
[Unit]
Description=Kelvin
ConditionPathExists=/opt/kelvin/kelvin
After=network.target

[Service]
Type=simple
User=kelvin
Group=kelvin

Restart=always
RestartSec=10
StartLimitInterval=60s

WorkingDirectory=/opt/kelvin
ExecStart=/opt/kelvin/kelvin

[Install]
WantedBy=multi-user.target

```
```
# Start, then press hue button. Restart if necessary
sudo systemctl start kelvin

# Start on boot
sudo systemctl enable kelvin

# Confirm status
sudo systemctl status kelvin

# Clean up
rm /tmp/kelvin-arm.tar.gz

# Edit config to taste
sudo -u kelvin -e /opt/kelvin/config.json
sudo systemctl restart kelvin

# Read Logs
journalctl -fu kelvin.service
```

**Comment by stefanwichmann** [comment]
Nice! Thanks for the input. This will definitely improve the installation process for linux users!

I absolutely plan to integrate this. Feel free to create a PR to speed things up. :-)

**Comment by sirwolfgang** [comment]
I ported this over and updated the service file in #55

**Comment by stefanwichmann** [comment]
Released with version 1.2.0
